### PR TITLE
Swift Pods + Frameworks as defaults

### PR DESCRIPTION
**Pull request by orta**
Asks an extra question at the beginning about whether you want to make a Swift framework.

### CODE REVIEW

File: setup/ConfigureiOS.rb [hunk 1]
```diff
@@ -21,26 +21,25 @@ def perform
...
+          configurator.add_line_to_pch "@import Expecta;"
 
-          configurator.set_test_framework("specta")
+          configurator.set_test_framework("specta", "m")
 
         when :kiwi
           configurator.add_pod_to_podfile "Kiwi"
-          configurator.add_line_to_pch "#import <Kiwi/Kiwi.h>"
-          configurator.set_test_framework("kiwi")
+          configurator.add_line_to_pch "#import Kiwi;"
```
Comment (segiddins) [inline setup/ConfigureiOS.rb]: `@import`

File: setup/ConfigureSwift.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+      Pod::ProjectManipulator.new({
+        :configurator => @configurator,
+        :xcodeproj_path => "templates/swift/Example/PROJECT.xcodeproj",
+        :platform => :ios,
+        :remove_demo_project => (keep_demo == :no),
+        :prefix => ""
+      }).run
+
+      `mv ./templates/swift/* ./`
```
Comment (segiddins) [inline setup/ConfigureSwift.rb]: please use `FileUtils` for moving files

File: setup/TemplateConfigurator.rb [hunk 3]
```diff
@@ -132,19 +140,22 @@ def add_line_to_pch line
...
+    def set_test_framework(test_type, extension)
+      content_path = "setup/test_examples/" + test_type + "." + extension
+      folder = extension == "m" ? "ios" : "swift"
+      tests_path = "templates/" + folder + "/Example/Tests/Tests." + extension
       tests = File.read tests_path
       tests.gsub!("${TEST_EXAMPLE}", File.read(content_path) )
       File.open(tests_path, "w") { |file| file.puts tests }
     end
-
+
```
Comment (segiddins) [inline setup/TemplateConfigurator.rb]: trailing whitespace

**Comment by orta** [comment]
Thanks @segiddins

**Pull request merged**